djangocms_text_ckeditor/forms.py

```diff
@@ -43,7 +43,8 @@ def get_child_plugins(self):
     def render_plugin(self, request):
         plugin = self.cleaned_data['plugin']
         context = RequestContext(request)
-        rendered_content = plugin.render_plugin(context)
+        context['request'] = request
```

Comment:
Django does not apply context processors until the template is rendered.
This means that I can't access context['request'] prematurely and so I need to set it explicitly